### DISCUSSION TITLE
Repository Github untuk upload RMarkdown

### DISCUSSION THREAD

**Discussion by ChristoNindyo** [teamalgoritma/community#217] [Q&A] [answered]
Salam, saya ingin bertanya perihal Repository Github untuk upload RMarkdown.

1. Apa saja yang wajib ada di repository Github untuk kita upload hasil RMardown ?
2. Apa perlu file rmd, html, ataupun csv diikutsertakan ?

Terima kasih sebelumnya.

**Comment by idoali** [comment, marked as answer]
Selamat Pagi Pak

Untuk mengupload RMarkdown ke GitHub yang dibutuhkan hanya file RMarkdown itu saja pak. Di RMarkdown ini bapak akan menampilkan hasil analisis dan visualisasi jadi pastikan juga RMarkdown yang terupload sudah di run semua. Semoga jawabannya dapat membantu pak

**Reply by ChristoNindyo** [reply]
oke, terima kasih, pak.
